Third party apps portal

omkarghaisas opened the pull request:
### Added OAuth Clients
![oauth_clients](https://user-images.githubusercontent.com/30148373/28571627-69fe0414-7161-11e7-91e8-99905ec5c3a2.png)

### Manage third party app from profile
![manage_third_party_apps](https://user-images.githubusercontent.com/30148373/28571736-ecd761c8-7161-11e7-93c1-3102f884a666.png)

### No Active Sessions - not signed in any third party app
![no_active_sessions](https://user-images.githubusercontent.com/30148373/28571631-74d28504-7161-11e7-95fa-ea1254973000.png)

### After  logging in from multiple Apps / devices
![active_sessions](https://user-images.githubusercontent.com/30148373/28571657-90e12106-7161-11e7-92b3-98499877646c.png)

### Confirmation screen 
![revoke_app_acess](https://user-images.githubusercontent.com/30148373/28571757-fecd07fc-7161-11e7-8951-b99ac073647b.png)

### Screen after app removed
![app_removed](https://user-images.githubusercontent.com/30148373/28571670-a1db3050-7161-11e7-8d49-87bcb59cdae1.png)

omkarghaisas committed:
Added third party apps portal page stub

omkarghaisas committed:
[WIP] third party apps portal page

omkarghaisas committed:
Merge branch 'develop' of https://github.com/frappe/frappe into third_party_apps_portal

omkarghaisas committed:
Added portal page third party apps

Added page to manage OAuth 2.0 active sessions

omkarghaisas committed:
[Fix] Typo me.html

rmehta commented:
@omkarghaisas nice!

Instead of the "x", just make a button element called "Revoke". In Frappe/ EPRNext we tend to use named buttons instead of icons / symbols.

Also check Codacy

omkarghaisas committed:
frappe/www/third_party_apps.

omkarghaisas committed:
Added Revoke button

Merge branch 'develop' of https://github.com/frappe/frappe into third_party_apps_portal

omkarghaisas commented:
After adding revoke button
![final screen app menu](https://user-images.githubusercontent.com/30148373/28613633-8a7aa8f0-7210-11e7-99db-c6f3af74bbe2.png)

rmehta commented:
👍 maybe a separate column for when they were last logged in, so it looks more aligned.

omkarghaisas committed:
[Fix] Added column for last log in

omkarghaisas commented:
[FIX]Added column right for last logged in

![final third party apps 2](https://user-images.githubusercontent.com/30148373/28617070-8f7d808c-721c-11e7-93aa-9a5abd21f535.png)
